Document/0x04f-Testing-Network-Communication.md

```diff
@@ -108,14 +108,150 @@ If that's the case, you are now able to see the complete network traffic that is
 
 > Man-in-the-middle attacks work against any device and operating system as the attack is executed on OSI Layer 2 through ARP Spoofing. When you are MITM you might not be able to see clear text data, as the data in transit might be encrypted by using TLS, but it will give you valuable information about the hosts involved, the protocols used and the ports the app is communicating with.
 
-#### Span Port / Port Forwarding
+#### Simulating a Man-in-the-Middle Attack with an Access Point
 
-As an alternative to a MITM attack with bettercap, a Wifi Access Point (AP) or router can also be used instead. The setup requires access to the configuration of the AP and this should be clarified prior to the engagement. If it's possible to reconfigure you should check first if the AP supports either:
+##### Network Setup
+
+Simple way to simulate a man-in-the-middle (MITM) attack is to configure network where all packets between devices and the target network are going through your machine. In mobile world, this is achieved by placing Access Point between mobile devices and your machine. Your machine is a router and an Access Point.
```

Comment:
```suggestion
A simple way to simulate a man-in-the-middle (MITM) attack is to configure a network where all packets between the devices in scope and the target network are going through your machine. In a mobile penetration test, this can be achieved by using an Access Point the mobile devices and your machine are connected to. Your machine is then becoming a router and an Access Point.
```